Skype/Trusted-Application-API/samples/QuickStartSamples/AudioVideoIVRSample.sln

```diff
@@ -0,0 +1,38 @@
+﻿
```

Comment:
Following solutions are to be kept in prerelease branch only:

1. AudioVideoIVRSample
2. AvBridgeSample
3. AvBridgeToSipUriSample
4. CallForwardSample
5. CallTransferSample
6. TrustedAudioVideoMeeting
7. TapSamples

It appears to me that you merged prerelease branch into your local branch. Deleting these solutions on your local branch will only add to the mess as then merging master into prerelease will remove these samples from prerelease branch also.

Can you please get a clean branch off master and copy your new solution there? Update this pull request with the new branch. If you need help with this, please ping/email me.